src/config/api.js

```diff
@@ -0,0 +1,27 @@
+import { create } from 'apisauce';
+import { CamelcaseSerializer, SnakecaseSerializer } from 'cerealizr';
+
+const api = create({
+  baseURL: process.env.REACT_APP_BASE_API_URL,
+  headers: {
+    'Content-Type': 'application/json',
+    Accept: 'application/json'
+  }
+});
+
+const serializer = new CamelcaseSerializer();
+const deserializer = new SnakecaseSerializer();
+
+api.addResponseTransform(response => {
+  if (response.data) {
+    response.data = serializer.serialize(response.data);
+  }
+});
+
+api.addRequestTransform(request => {
+  if (request.data) {
+    request.data = deserializer.serialize(request.data);
+  }
```

Comment:
this is for request, and i think its different here, i might be wrong

Reply:
mm, Im using two diferents objects const serializer = new CamelcaseSerializer();
const deserializer = new SnakecaseSerializer();

Reply:
I see this well, you are using CamelcaseSerializer for the response, and SnakecaseSerializer for the request

Reply:
im refering to the `request` not about the serializer, but if it works, its ok